plugin/pkg/scheduler/schedulercache/cache.go

```diff
@@ -0,0 +1,232 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package schedulercache
```

Comment:
What do you think about renaming this package to just "cache"?
I think that duplication "scheduler" here (since we already have it in "plugin/pkg/scheduler" is unnecessary.

This should be similar to "pkg/client/cache" (not "pkg/client/clientcache").